### PR TITLE
#161041638 Create login page and welcome panel

### PR DESCRIPTION
**Pull request by codeMarble254**
#### What does this PR do?
This PR creates the login page for this application
#### Description of Task to be completed?
The main tasks to be completed are:\
Creation of a login form\
Creation of a welcome section on the page
#### How should this be manually tested?
To test this page, clone my repository using the link ```https://github.com/codeMarble254/Store-Manager.git```
Then open the terminal and change the current directory to ```Store-Manager```\
Switch the current branch to``` ft-login-UI-161041638``` by typing ```git checkout ft-login-UI-161041638```\
Finally, launch ```index.html```
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
#161041638
#### Screenshots (if appropriate)
screenshot from 2018-10-10 02-41-35

#### Questions:
How does the UI look? Any improvements i can make?

### CODE REVIEW

**Comment by Georgeygigz** [comment]
Nyc interface

**Comment by Hesbon5600** [comment]
Great UI.

**Comment by Krafty-Coder** [comment]
Otherwise great UI and perfect choice of colors...

**Comment by mimipeshy** [comment]
Great UI , easy to navigate

**Comment by sharkdevs** [comment]
hey check how you have named that PR.. Its not conventional.. check it here https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices.. Otherwise Its ok

File: UI/index.html [hunk 1]
```diff
@@ -1,42 +1,49 @@
...
+            </form>
 
 
         </div>
     </section>
+    <footer>
+        &copy;CodeMarble
+    </footer>
 </body>
+<script src="static/main.js"></script>
```
Comment (Krafty-Coder) [inline UI/index.html]: This should be before the `body` tag

Comment (codeMarble254) [reply UI/index.html]: Ok. Let me change that

**Comment by Paulstar200** [comment]
Amazing!!!!

**Pull request merged**